test/e2e/strip_randomness.ml

```diff
@@ -0,0 +1,37 @@
+let standardise_filesep =
+  let re = Re.compile (Re.str Filename.dir_sep) in
+  Re.replace_string ~all:true re ~by:"/"
```

Comment:
Aren't you afraid this approach is a bit aggressive? I remember receiving notification about sanitizing test names so they might not contain `/` anymore in which case it's not much of an issue.

I don't think that's a major issue as there's probably no real harm in swapping those.

One thing worth considering is to proceed with this replacement based on the value of `Sys.os_type` or `Sys.win32` for instance.

Let me know what you think!

Reply:
I think this is currently safe, since these characters are banned in test names. In future, they may not be, but that would have to come with a more substantial revision of the structure of log outputs from Alcotest. 

I'm not particularly worried by this solution for now, but could be convinced otherwise -- bizarre characters in test names shouldn't be making their way onto the file-system whatever we do.

Reply:
It seems fine to me for now! Let's go with that!